src/lib/Asset.ts

```diff
@@ -262,14 +268,18 @@ class Asset {
           this.#config.pool.location === "uni"
             ? UNISWAP_PAIR_DATA
             : SUSHISWAP_PAIR_DATA;
-        const poolData = await request(endpoint, query, {
+        // eslint-disable-next-line
```

Comment:
This was an implicit any before so I just disabled eslint for this line for now.